infrastructure/templates/screener/main.bicep

```diff
@@ -23,12 +23,12 @@ param resourceTags {
 @description('Provides access to resources for specific IP address ranges used for service maintenance.')
 param maintenanceIpRanges IpRange[] = []
 
-@description('Specifies the Application (Client) Id of a pre-existing App Registration used to represent the Screener Function App.')
-param screenerAppRegistrationClientId string = ''
+// @description('Specifies the Application (Client) Id of a pre-existing App Registration used to represent the Screener Function App.')
+// param screenerAppRegistrationClientId string = ''
 
-@description('Specifies the principal id of the Azure DevOps SPN.')
-@secure()
-param devopsServicePrincipalId string = ''
+// @description('Specifies the principal id of the Azure DevOps SPN.')
+// @secure()
+// param devopsServicePrincipalId string = ''
```

Comment:
I think these unused variables could be removed now?
Some are commented out below, but they can probably be removed too?